momento-provider/src/main/java/com/google/code/ssm/providers/momento/CachedObjectWrapper.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2014-2019 Jakub Białek
```

Comment:
We have to manually copy over our own version since we can't pull directly from the file in the repo

Reply:
Sorry I'm not sure I understand that reply, can you elucidate? I don't know where this file was copied from nor do I understand why it needed to be copied instead of referenced.

Reply:
This was copied from https://github.com/ragnor/simple-spring-memcached/blob/master/spymemcached-provider/src/main/java/com/google/code/ssm/providers/spymemcached/CachedObjectWrapper.java

`spymemcached` is considered a separate module/dependency. Instead of taking the entire module as a dependency I figured we could just copy it over so there's no namespace conflicts